oval_graph/converter.py

```diff
@@ -65,12 +77,20 @@ def to_JsTree_dict(self):
                    tag=self.get_tag(),
                    comment=self.get_comment()),
                "icon": icons['icon'],
-               "state": {"opened": True}}
+               "state": {"opened": self._hide_node(option)}}
         if self.tree.children:
-            out['children'] = [Converter(child).to_JsTree_dict()
+            out['children'] = [Converter(child).to_JsTree_dict(option)
                                for child in self.tree.children]
         return out
 
+    def _hide_node(self, option):
+        value = self.tree.evaluate_tree()
+        if value is None:
+            value = self.tree.value
+        if value in self.HIDE_PASS_TESTS[option].keys():
+            return self.HIDE_PASS_TESTS[option][value]
+        return True
```

Comment:
I'm not sure if I understand it. Correct me if I'm wrong. Probably, it means that:
* if user doesn't provide `--hide-pass-tests` it always returns True.
* if user provides  `--hide-pass-tests`:
  * if the node is "true" returns False
  * otherwise returns True

I think it can be rewritten to make the code more readable in this manner, and you won't need the dictionary.

Also, the function name `_hide_node` is confusing. Based on the name I would expect that when it returns True it means that the node will be hidden, but it seems it's the opposite - when it returns False the node will be hidden. So I recommend either swapping True and False return values or renaming the function to `_show_node`.